.pre-commit-config.yaml

```diff
@@ -41,13 +41,20 @@ repos:
         args:
           - --rcfile=.pylintrc
 
-  # https://pre-commit.com/#repository-local-hooks
-  - repo: local
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.991
     hooks:
       - id: mypy
-        name: mypy
-        entry: mypy
-        language: system
-        types: [python]
-        files: \.py$
         exclude: ^setup.py|^tests/|^vendor/|^astacus/proto/
+        additional_dependencies:
+          - types-PyYAML>=6.0.12.2
+          - types-requests>=2.28.11.4
+          - types-tabulate>=0.9.0.0
+          - types-urllib3>=1.26.25.3
+          - typing_extensions>=3.10.0
+          - pydantic<2.0.0,>=1.6.2
```

Comment:
The version used here should be exact match with what we use elsewhere.

Reply:
Will double check that the constraints are correct.

Reply:
Doubled checked and everything's pinned to the same versions.